app/uk/gov/hmrc/individualsincomeapi/domain/SaSubmission.scala

```diff
@@ -71,4 +71,16 @@ object SaTaxReturnSummaries {
       TaxYear.fromEndYear(desSAIncome.taxYear.toInt),
       desSAIncome.returnList.map(sa => SaTaxReturnSummary(sa.utr, sa.incomeFromSelfAssessment.getOrElse(0.0))))
   }
+}
+
+case class SaAnnualTrusts(taxYear: TaxYear, trusts: Seq[SaAnnualTrust])
+case class SaAnnualTrust(utr: SaUtr, trustIncome: Double)
```

Comment:
can we follow the same pattern than the others?
SaAnnualTrusts and SaTrustsIncome

Reply:
👍